<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpsSatellites</class>
 <widget class="QWidget" name="OpsSatellites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>575</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>View options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QCheckBox" name="kcfg_ShowSatellites">
            <property name="text">
             <string>Show satellites</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">satelliteButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_ShowVisibleSatellites">
            <property name="text">
             <string>Show only visible satellites</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">satelliteButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="kcfg_ShowSatellitesLabels">
            <property name="text">
             <string>Show labels</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">satelliteButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="kcfg_DrawSatellitesLikeStars">
            <property name="text">
             <string>Draw satellites like stars</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">satelliteButtonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>List of satellites</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="FilterEdit">
            <property name="placeholderText">
             <string>Search satellites</string>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="UpdateTLEButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Update TLEs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeView" name="SatListTreeView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="satelliteButtonGroup"/>
 </buttongroups>
</ui>
